packages/odyssey-react-mui/src/OdysseyTranslationProvider.tsx

```diff
@@ -44,14 +44,33 @@ export type OdysseyTranslationProviderProps<
   translationOverrides?: TranslationOverrides<SupportedLanguages>;
 };
 
+const formatLanguageCodeToHyphenated = <SupportedLanguages extends string>(
+  languageCode: OdysseyTranslationProviderProps<SupportedLanguages>["languageCode"]
+) => languageCode?.replace(/_/g, "-");
+
 export const OdysseyTranslationProvider = <SupportedLanguages extends string>({
   children,
   languageCode,
   translationOverrides,
 }: OdysseyTranslationProviderProps<SupportedLanguages>) => {
   useEffect(() => {
+    const languageCodeIncludesUnderscore = languageCode?.includes("_");
+
+    const normalizedLanguageCode = languageCodeIncludesUnderscore
+      ? formatLanguageCodeToHyphenated<SupportedLanguages>(languageCode)
```

Comment:
Can we simplify this?

```suggestion
    const normalizedLanguageCode = formatLanguageCodeToHyphenated<SupportedLanguages>(languageCode)
```

We don't need to do redundant if-checks if they don't change anything.